js/components/operatorSignIn/attestation.tsx

```diff
@@ -16,14 +16,15 @@ export const Attestation = ({
 }: {
   badge: string;
   employees: ApiResult<Employee[]>;
-  onComplete: () => void;
+  onComplete: (radio: string) => void;
```

Comment:
praise: passing the radio on complete instead of trying to keep the state updated in the parent is way better, good call on that.